cccl

```diff
@@ -39,6 +39,30 @@ case $MACHTYPE in
         ;;
 esac
 
+for arg in $@; do
+    if [ "$arg" == "-dumpmachine" ] || [ "$arg" == "-dumpmachinefull" ]; then
```

Comment:
Thanks for the suggestion. I will fix it to make it go along with other parts of the code as well.

Reply:
I prefer the original, much simpler and easier to follow and probably more portable and possibly more portable if using `=`. @vadz is right, we do use `if test` everywhere, I think because we originally used sh instead of bash. Anyway, I'm no bash expert and don't really understand portability subtleties in bash except that what we currently using is meant to be more portable. Anyway, happy to be defer to some bash experts here.